### ISSUE TITLE
Autowhatever require specific props ?? 

### ISSUE THREAD

**Issue by thomaslc66** [moroshko/react-autosuggest#738]
## Are you reporting a bug?
Don't know, but seems like an odd behavior

I've not set multiSection={true}
My code look like this :
````
        <Autosuggest
          suggestions={suggestions}
          onSuggestionsFetchRequested={onSuggestionsFetchRequested}
          onSuggestionsClearRequested={onSuggestionsClearRequested}
          getSuggestionValue={this.getSuggestionValue}
          renderSuggestion={renderSuggestion}
          onSuggestionSelected={this.onSuggestionSelected}
          inputProps={inputProps}
        />
````

But in the console i have those 2 messages, but i don't need multiSection and even if i set multiSection={false} the error is still the same

![image](https://user-images.githubusercontent.com/9827392/80316208-9cc6bd00-87fc-11ea-9a37-6413d7747da3.png)

**Comment by masbaehr** [comment]
Same issue recently needs to be fixed please

**Comment by aberezkin** [comment]
Working on it.

**Comment by tpenaranda** [comment]
As a workaround @thomaslc66, adding "empty" functions on `renderSectionTitle` and `getSectionSuggestions` props did the trick for me.

```
<Autosuggest
  ...
  renderSectionTitle={() => {}}
  getSectionSuggestions={() => {}}
/>
```

**Comment by jscul** [comment]
tpenaranda's suggestion didn't fix the issue. Even if I put `multiSection={false}`. No luck.

**Comment by adriancorcoran** [comment]
@tpenaranda Solution worked also, empty functions are the way to go to solve this for now

**Comment by aberezkin** [comment]
Fixed in v10.0.2
